float64 center_x
float64 center_y
float64 v_x
float64 v_y
float64 size_x
float64 size_y